### DISCUSSION TITLE
How can i add new store?(RU)

### DISCUSSION THREAD

**Discussion by vadimkaxd** [jef/streetmerchant#1911] [Help]
In Russia there is not a single program that checks the status of a store, of course I'm not a strong programmer, but for sure everything is organized, some kind of cfg or something else? Or I can read somewhere a guide, thanks in advance.
upd: I search all what want, close disc.

**Comment by gutshotexist** [comment]
Hi, Vadim. Как связаться с тобой?

**Comment by wppah** [comment]
> 
> 
> In Russia there is not a single program that checks the status of a store, of course I'm not a strong programmer, but for sure everything is organized, some kind of cfg or something else? Or I can read somewhere a guide, thanks in advance.
> upd: I search all what want, close disc.

what did u do i need help adding store

**Reply by vadimkaxd** [reply]
> > In Russia there is not a single program that checks the status of a store, of course I'm not a strong programmer, but for sure everything is organized, some kind of cfg or something else? Or I can read somewhere a guide, thanks in advance.
> > upd: I search all what want, close disc.
> 
> what did u do i need help adding store

don't try buy someone by this programm it's losing time, in russia users use autobuy and u can't 
On RU
Можешь не пытаться что-то словить челы тупа юзают фаст покупку проверенно прям во время слива(ловил на озоне в мвидео/эльдорадо что то типа защиты)
так что нет смысла тратить на это время и вообще забей лучше пока автор не добавит автопокупку софт бесполезный ну в России уж точно но фидбека от автора нет и видимо не будет, ну точно не в моих темах, примечаний.

**Comment by StanZha** [comment]
There is a guide on how to add new store:
https://jef.codes/streetmerchant/help/general/

App does not buy items intentionally. I believe it is not going to change
